docs/application/vstools/install.md

```diff
@@ -0,0 +1,170 @@
+<div id="toc-navigation">
+
+# Installing Visual Studio Tools for Tizen
+
+You need the following components on top of Visual Studio to make Visual Studio Tools for Tizen work:
+
+- VSIX
+
+  Visual Studio extension for Tizen packaging
+
+- Tizen Baseline SDK
+
+  The SDK is shared with the Tizen Studio, and it supports tools (such as Certificate Manager, Device Manager, Emulator, SDB, and on-demand rpm) for developing Tizen .NET applications.
+
+
+## Prerequisites
+
+To work with Visual Studio Tools for Tizen, your computer must have:
+
+- At least 1.5 GB of available disk space
+- Visual Studio 2017
+
+  Visual Studio Tools for Tizen works with all Visual Studio variations, including Community. Installing or re-installing Visual Studio with .NET desktop development and .NET Core cross-platform development toolsets is recommended.
+
+  ![Visual Studio prerequisites](media/prerequisite-vs.png)
+
+- Java Development Kit (JDK) 8
+
+  JDK 8 is needed to use the Tizen Baseline SDK. OpenJDK is not supported.
+
+  Download the JDK 8 from the official [Oracle Web site](http://www.oracle.com/technetwork/java/javase/downloads/index.html). Select the appropriate platform for your hardware architecture and Windows&reg; version. Run the downloaded executable file and follow the displayed instructions.
+
+
+### Emulator Requirements
+
+The Tizen emulator for Visual Studio has the same requirements as the emulator in the Tizen Studio. To check the detailed hardware and software requirements for the Tizen emulator, see [Emulator Requirements](../tizen-studio/setup/prerequisites.md#emulator).
+
+- Intel&reg; Hardware Acceleration Execution Manager (Intel&reg; HAXM) speeds up the Tizen emulation on Intel-VT-enabled systems. The Intel&reg; HAXM installation is started automatically as part of the Visual Studio Tools for Tizen installation. For more information, see [Hardware Accelerated Execution Manager](../tizen-studio/setup/hardware-accelerated-execution-manager.md)
+
+- Make sure **Hyper-V** is disabled (in Windows&reg; 10 or higher):
+  1. In the **Start** menu, select **Programs and Features**.
+
+   ![Programs and Features](media/cs_prerequisite01-250x401.png)
+
+  2. Select **Turn Windows features on or off**.
+  3. Disable **Hyper-V** and click **OK**.
+
+   ![Disable Hyper-V](media/cs_prerequisite-disable-hiperv.png)
+
+  4. Reboot the computer.
+
+
+## Visual Studio Tools for Tizen Installation {#install}
```

Comment:
Is it ok to use in this form? `{#install}`

Reply:
done